SPECS-EXTENDED/xhtml1-dtds/xhtml1-dtds.spec

```diff
@@ -1,18 +1,17 @@
 %global date    20020801
 
+# W3C Software License for DTDs etc:
+# http://www.w3.org/Consortium/Legal/IPR-FAQ-20000620#DTD
 Name:           xhtml1-dtds
 Version:        1.0
-Release:        20020802%{?dist}
+Release:        20020803%{?dist}
```

Comment:
Ah, now I see what you meant about the release and date for this spec. For some reason I thought the version was the date incremented by one. Yeah, this is not great but fixing this is still out of scope for this PR.